File: rd-kt/rd-framework/src/jvmTest/kotlin/com/jetbrains/rd/framework/test/cross/util/FileSystem.kt
package com.jetbrains.rd.framework.test.cross.util

import java.io.File

val rdTmpDir: File = File(System.getProperty("java.io.tmpdir"), "rd")
    get() {
        field.mkdirs()
        return field
    }

val portFile = File(rdTmpDir, "port.txt")
    get() {
        rdTmpDir.mkdirs()
        return field
    }

File: rd-net/Test.RdCross/Util/FileSystem.cs
using System.IO;

namespace Test.RdCross.Util
{
    public static class FileSystem
    {
        static string RdTmpDir => Path.Combine(Path.GetTempPath(), "rd");

        internal static string PortFile => Path.Combine(RdTmpDir, "port.txt");
    }
}